src/pages/mobile/SettingsPasswordSet.vue

```diff
@@ -32,23 +20,34 @@
     >
       <PasswordPurpose />
       <AeInputPassword
+        v-if="isWalletEncrypted"
         v-model="password"
         v-validate="'required|min:4'"
-        autofocus
-        autocomplete="new-password"
+        :autofocus="isWalletEncrypted"
+        autocomplete="current-password"
         :error="errors.has('password')"
         :footer="errors.first('password')"
         name="password"
+      />
```

Comment:
```suggestion
      >
        <template slot="footer">
          <template v-if="errors.has('password')">
            {{ errors.first('password') }}
          </template>
          <template v-else-if="wrongPassword">
            Wrong password, try again
          </template>
        </template>
      </AeInputPassword>
```
In general i propose to show user if there is a wrong password entered.